src/ORM/DataObject.php

```diff
@@ -940,6 +949,50 @@ public function i18n_plural_name()
         return _t(static::class . '.PLURALNAME', $this->plural_name());
     }
 
+    /**
+     * Get description for this class
+     * @return null|string
+     */
+    public function classDescription()
+    {
+        return static::config()->get('class_description', Config::UNINHERITED);
+    }
+
+    /**
+     * Get localised description for this class
+     * @return null|string
+     */
+    public function i18n_classDescription()
+    {
+        $placeholder = 'PLACEHOLDER_DESCRIPTION';
```

Comment:
Unfortunately not, because `_t()` does a truthy check against `$default`
https://github.com/silverstripe/silverstripe-framework/blob/1f2ae2a201111f366c9442fcca5857379c69554d/src/i18n/i18n.php#L169-L171

Reply:
Done